src/app/(dashboard)/teams/[teamId]/page.tsx

```diff
@@ -165,7 +176,60 @@ export default function Page({ params }: PageProps) {
               </div>
             ) : (
               <div>
-                <div className="pt-6 grid gap-4 grid-cols-1 md:grid-cols-2 lg:grid-cols-3">
+                <div className="grid gap-4 md:grid-cols-2 lg:grid-cols-3">
+                  <Card>
+                    <CardHeader className="flex flex-row items-center justify-between space-y-0 pb-2">
+                      <CardTitle className="text-sm font-medium">
+                        Total team CRED
+                      </CardTitle>
+                      <Icons.pizza></Icons.pizza>
+                    </CardHeader>
+                    <CardContent>
+                      <div className="text-2xl font-bold">
+                        {overview?.totalScore.toFixed(2)}
+                      </div>
+                      <p className="pt-1 text-xs text-muted-foreground">
+                        Total CRED earned for team
+                      </p>
+                    </CardContent>
+                  </Card>
+
+                  <Card>
+                    <CardHeader className="flex flex-row items-center justify-between space-y-0 pb-2">
+                      <CardTitle className="text-sm font-medium">
+                        Team CRED earned this week
+                      </CardTitle>
+                      <Icons.pizza></Icons.pizza>
+                    </CardHeader>
+                    <CardContent>
+                      <div className="text-2xl font-bold">
+                        {overview?.weekScore.toFixed(2)}
+                      </div>
+                      <p className="pt-1 text-xs text-muted-foreground">
+                        +{overview?.weekGrowth.toFixed(2)}% from last week
+                      </p>
+                    </CardContent>
+                  </Card>
+
+                  <Card>
+                    <CardHeader className="flex flex-row items-center justify-between space-y-0 pb-2">
+                      <CardTitle className="text-sm font-medium">
+                        Total contributors
+                      </CardTitle>
+                      <Icons.pizza></Icons.pizza>
+                    </CardHeader>
+                    <CardContent>
+                      <div className="text-2xl font-bold">
+                        {overview?.totalContributors}
+                      </div>
+                      <p className="pt-1 text-xs text-muted-foreground">
+                        All contributors on all your teams
```

Comment:
This copy is wrong, should be All contributors on this team, or on your team, something like that